src/marlin_stubs/pause/G27.cpp

```diff
@@ -33,9 +33,8 @@
  */
 void GcodeSuite::G27() {
     // Don't allow nozzle parking without homing first
-    if (axis_unhomed_error())
-        return;
-    nozzle.park(parser.ushortval('P'));
+    if (!axis_unhomed_error())
```

Comment:
why did U edit this code?
It does same thing as before.

Reply:
I was just trying to get one bug and did not return it

Reply:
than please revert this, fewer changes == better